src/MCControl.cpp

```diff
@@ -421,6 +381,20 @@ RTC::ReturnCode_t MCControl::onInitialize()
   bindParameter("is_enabled", controller.running, "0");
   bindParameter("is_simulation", m_is_simulation, "0");
 
+  // Open IOB
+  if(!m_is_simulation)
```

Comment:
I think it's too early to check this here because we set `is_simulation` later in our files. Maybe it's better to do it during the init part or simply the first time that we need to open the IOB for a call

Reply:
Really? How come it worked on the real robot in that case?

> Maybe it's better to do it during the init part or simply the first time that we need to open the IOB for a call

The first time that we make a IOB call is in the init part (where we check the timestep). So let's do `open_iob` there?

Reply:
> Really? How come it worked on the real robot in that case?

Because `m_is_simulation` defaults to `false`

> The first time that we make a IOB call is in the init part (where we check the timestep). So let's do open_iob there?

I think that's ok. There's no way to other IOB related calls before

Reply:
Okay, I fixed.